crates/service-client/Cargo.toml

```diff
@@ -19,15 +19,22 @@ schemars = { workspace = true, optional = true }
 
 arc-swap = { workspace = true }
 base64 = { workspace = true }
+bytes = { workspace = true }
 bytestring = { workspace = true }
 futures = { workspace = true }
-http-serde = { workspace = true }
 humantime = { workspace = true }
-hyper = { workspace = true }
-hyper-rustls = { workspace = true }
-h2 = { version = "0.3.20" }
+
+h2 = "0.4.5"
+hyper = { version = "1.4.1", features = ["http1", "http2", "client"] }
+hyper-util = { version = "0.1.6", features = ["client-legacy"]}
+http = "1.1.0"
+http-serde = "2.0.0"
+http-body-util = "0.1.2"
+hyper-rustls = { version = "0.27.2", features = ["http2"] }
+tower-service ={ version = "0.3" }
+rustls = "0.23.11"
```

Comment:
We try to sort dependencies alphabetically w/o trying to create logical groups.

Reply:
do we have a tool for doing this?

Reply:
no, it's manual/best effort.

Reply:
this doesn't make it any easy :smile:

Reply:
True. The benefit is that this a pattern that we can somehow consistently apply throughout the code base.